std_msgs/Int32 cmode
std_msgs/Float64 feature_1
std_msgs/Float64 feature_2
std_msgs/Float64 feature_3
std_msgs/Float64 feature_4
geometry_msgs/Vector3 target_pose
geometry_msgs/Vector3 target_vel

